tensorboard/webapp/runs/views/runs_table/regex_edit_dialog.ng.html

```diff
@@ -14,23 +14,38 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
+<h1 mat-dialog-title>Color runs by regex</h1>
+
 <mat-dialog-content>
+  <p>Enter a regex with capturing groups to match against run names:</p>
   <mat-form-field>
     <input
       matInput
       #regexStringInput
-      placeholder="(\w+/).+"
       value="{{ regexString }}"
       (keydown.enter)="onEnter($event.target.value)"
       cdkFocusInitial
     />
   </mat-form-field>
 </mat-dialog-content>
 
-<div mat-dialog-actions>
+<div class="example-details">
+  <p>
+    Each matching run will be assigned a color based on the "key" formed by its
+    matches to the capturing groups. <br />
+    <a href="#" (click)="fillExample('(train|eval)')">
```

Comment:
This is semantically a button, not an anchor element that takes you to a different page. Please use semantic element, `<button>` here. You don't have to use mat-button for this.

Reply:
Do you mean use `<button>` but it doesn't have to look like a button?  I suggested making it appear like a link earlier since I think something more text-oriented would fit better in an inline/paragraph context.  But I guess it could still use `<button>` like these link-lookalikes here on Github do:

![Screen Shot 2021-07-16 at 12 23 23 PM](https://user-images.githubusercontent.com/710113/125998687-6e365d71-bf13-4277-912c-8c266dde1c04.png)

If you mean something else maybe we should come to some consensus first before asking Jie-Wei to change it back and forth.

Reply:
> Do you mean use <button> but it doesn't have to look like a button? I suggested making it appear like a link earlier since I think something more text-oriented would fit better in an inline/paragraph context. But I guess it could still use <button> like these link-lookalikes here on Github do:

Correct. For accessibility and other purposes, you want button to use `<button>`. 

Yes, I am not objecting to your feedback of wanting to see a link-like UI for the button.